test/e2e/reboot.go

```diff
@@ -95,28 +114,44 @@ var _ = Describe("Reboot", func() {
 })
 
 func testReboot(c *client.Client, rebootCmd string) {
+	// These tests requires SSH to nodes, so the provider check should be identical to there
+	// (the limiting factor is the implementation of util.go's getSigner(...)).
+
+	// Cluster must support node reboot
+	SkipUnlessProviderIs(providersWithSSH...)
```

Comment:
nit: this seems redundant with the check on L55 (though it's written differently there)